code/modules/mob/living/carbon/carbon.dm

```diff
@@ -598,4 +598,36 @@ var/const/GALOSHES_DONT_HELP = 4
 		if(locate(/obj/item/device/assembly/health) in src)
 			stat(null, "Health: [health]")
 
-	add_abilities_to_panel()
\ No newline at end of file
+	add_abilities_to_panel()
+
+/mob/living/carbon/proc/add_screams(var/list/screams)
+	if(!alternate_screams)
+		alternate_screams = list()
+
+	if(!screams || screams.len == 0)
+		return
+
+	for(var/S in screams)
+		if(!locate(S) in alternate_screams)
+			alternate_screams += S
+
+/mob/living/carbon/proc/reindex_screams()
+	src.alternate_screams = list()
```

Comment:
Necessary to who? Not to the code,but if you mean to the reader,meh. If you are referring to the head alternate scream you have to use head.alternate_screams so